src/Cli/dotnet/SlnFileFactory.cs

```diff
@@ -11,6 +11,9 @@ namespace Microsoft.DotNet.Cli;
 
 public static class SlnFileFactory
 {
+    public static readonly string[] DefaultPlatforms = new[] { "Any CPU", "x64", "x86" };
```

Comment:
Is there a reason arm64 is not included?

Reply:
Yes, this was moved from src/Cli/dotnet/Commands/Solution/Add/SolutionAddCommand.cs

The reason it was not included there is because when adding support for SLNX we wanted to maintain backwards compatibility with SLN behavior, and the behavior at the time was only adding these values

Reply:
We were very strict with changes when moving to vs-solutionpersistence, but I don't see any reason why we couldn't add arm64 in the future. These values are also only used for sln, not slnx